softperipheral/sQSPI/include/nrf54h20/sqspi_firmware.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SQSPI_FIRMWARE_H__
+#define SQSPI_FIRMWARE_H__
+#include "sqspi_firmware_v0.1.0.h"
```

Comment:
How about changing this to:
```
#if defined(NRF54L15_XXAA)
#include "nrf54l15/sqspi_firmware_v0.1.0.h"
#elif defined(NRF54H20_XXAA)
#include "nrf54h20/sqspi_firmware_v0.1.0.h"
#endif
```
and moving the file one level up (or maybe even place all of this next to `nrfx_qspi2.c`, I guess only this file is supposed to include this stuff)? This would simplify a bit the integration of the driver.